docs/ELRS_RECEIVERS.md

```diff
@@ -0,0 +1,63 @@
+# mLRS Documentation: ELRS Receivers #
+
+([back to main page](../README.md))
+
+## Supported ELRS Receivers ##
+
+As of April 2024, ELRS receivers with an ESP8285 MCU are experimentally supported by mLRS. The following targets are supported:
+
+- Generic 900
+- Generic 900 PA
+- Generic 2400
+- Generic 2400 PA
+
+To determine if your receiver hardware is supported, go to [ELRS Targets](https://github.com/ExpressLRS/targets/blob/master/targets.json) and find the layout file of your hardware.
+
+Note: ELRS receivers that use an ESP32 MCU are under active development.
+
+## Recommended ELRS Receivers ##
+
+The following receivers are recommended as they support >= 100 mW transmit power and have been validated by the mLRS team.
```

Comment:
maybe less provokative for other vendors:
The following receivers are good choices. They support >= 100 mW, are among the cheapest in their segment, and have been validated by the mLRS team.

Reply:
Will take this and tweak it slightly.